app/models/authenticator/base.rb

```diff
@@ -189,6 +189,9 @@ def find_by_principalname(username)
       [user, username]
     end
 
+    alias find_by_principalname lookup_by_principalname
+    Vmdb::Deprecation.deprecate_methods(self, :find_by_principalname => :lookup_by_principalname)
```

Comment:
are you able to get the deprecations back into the original files?

the goal was to split up the PRs so they cause fewer conflicts when back-porting PRs in the future. Having a single commit that touches all these files undoes that intent

Reply:
Finally, I am able to separate them into different commits ✌️Please check